streamz/dataframe/aggregations.py

```diff
@@ -167,18 +167,20 @@ def diff_iloc(dfs, new, window=None):
         List of dataframes to decay
     """
     dfs = deque(dfs)
-    dfs.append(new)
+    if len(new) > 0:
```

Comment:
~`not new.empty`~

^ it turns out that using `len` is consistently faster than using `empty`, I can't imagine why.